webpack.common.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         // Studio
         AssetsPage: './node_modules/@edx/studio-frontend/src/index.jsx',
         Import: './cms/static/js/features/import/factories/import.js',
+        AlertStatusBar: './lms/static/js/accessible_components/StatusBarAlert.jsx',
```

Comment:
Nit: this line shouldn't really be listed in the `// Studio` section; we're probably better off changing the next section header to a general `// LMS` and placing it there.

Reply:
Word up, makes sense. Will do.